### DISCUSSION TITLE
Install libstrongswan-extra-plugins by default

### DISCUSSION THREAD

**Discussion by vampywiz17** [linuxdeepin/developer-center#3305] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello there

Some days ago i try to create a IPsec VPN connection. Every time i get error that VPN is not established. I google a lot but not found any sollution.  but now, finally i found the problem!

It need to install this package:

libstrongswan-extra-plugins

and after it work like a charm!

My idea is that would be good that install this package default, because if you do this, ipsec work out of the box.

**Comment by BLumia** [comment]
cc @Zeno-sole
